### PR TITLE
Add support for ext x image tag

### PR DESCRIPTION
**Pull request by darkbaboon**
Hi,
This pull request add the support for EXT-X-IMAGE tag (also known as Thumbnails) #287 

 I made a rebase from the master, update my branch and resolve all the conflicts

### CODE REVIEW

File: m3u8/model.py [hunk 1]
```diff
@@ -1477,6 +1507,214 @@ def __str__(self):
...
+                'PATHWAY-ID=' + quoted(self.image_stream_info.pathway_id)
+            )
+        if self.image_stream_info.stable_variant_id:
+            image_stream_inf.append(
+                'STABLE-VARIANT-ID=' + quoted(self.image_stream_info.stable_variant_id)
+            )
+
+        return '#EXT-X-IMAGE-STREAM-INF:' + ','.join(image_stream_inf)
+
+class Tiles(BasePathMixin):
```
Comment (mauricioabreu) [inline m3u8/model.py:1688]: I think this class was already declared a few lines above

Comment (darkbaboon) [reply m3u8/model.py]: You are right ! I remove the duplicate code

**Comment by crvinay** [comment]
Any updates on this? I'd like to use these new tags if this PR is merged

**Comment by mauricioabreu** [comment]
Thank you for your contribution @darkbaboon

**Comment by mauricioabreu** [comment]
@crvinay merged ✅

**Pull request merged**